pkg/providers/tinkerbell/controlplane.go

```diff
@@ -34,7 +38,7 @@ func (p ControlPlane) Objects() []kubernetes.Object {
 }
 
 // ControlPlaneSpec builds a Tinkerbell ControlPlane definition based on an eks-a cluster spec.
-func ControlPlaneSpec(ctx context.Context, logger logr.Logger, client kubernetes.Client, clusterSpec *cluster.Spec) (*ControlPlane, error) {
+func ControlPlaneSpec(ctx context.Context, logger logr.Logger, client client.Client, clusterSpec *cluster.Spec) (*ControlPlane, error) {
```

Comment:
How does this affect cli flow? How can you make it reusable, unless we need to introduce a different method.

Reply:
This method is only used in controller, not cli